examples/netdev/demo_port_channel.pp

```diff
@@ -17,8 +17,8 @@
 class ciscopuppet::netdev::demo_port_channel {
   port_channel { 'port-channel100':
     ensure        => 'present',
-    id            => '100',
+    id            => 100,
     interfaces    => ['ethernet1/8', 'ethernet1/9'],
-    minimum_links => '3',
```

Comment:
Just double checking here but will the input validation handle both `'3'` and `3`?

Reply:
With the conversion to use resource_api it now has strict checking, so the netdev type for port channel defines it as an Integer, so unfortunately `3` and `'3'` are not the same and `'3'` would cause a validation error which is why we have updated the examples.

Reply:
Doesn't that mean that any customers that consume this will have to update all of their existing manifests if they are using a string representing an integer.  That concerns me a little.

Reply:
Yes it is something we are documenting that if they are moving from agent ~> agentless then they would need to update existing netdev resources.